doc/whats_new/v1.6.rst

```diff
@@ -243,6 +243,13 @@ Changelog
   right child node as the tree is traversed.
   :pr:`28268` by :user:`Adam Li <adam2392>`.
 
+:mod:`sklearn.frozen`
+.....................
+
+- |Feature| :class:`~sklearn.frozen.FrozenEstimator` is now introduced which allows
```

Comment:
I think this is hard to understand if you don't already know what frozen means in this context and if you do, you probably don't need more than the class name. Maybe we can improve this once we settle on the first line of the docstring of the class.

Reply:
improved.